pandas/tests/io/test_excel.py

```diff
@@ -1150,109 +1148,114 @@ def test_excel_sheet_by_name_raise(self, *_):
         with pytest.raises(xlrd.XLRDError):
             pd.read_excel(xl, "0")
 
-    def test_excel_writer_context_manager(self, *_):
+    def test_excel_writer_context_manager(self, frame, frame2, *_):
```

Comment:
An unintended consequence of the `*_` in the signature I think is that it makes all of the fixtures available within these functions which seems undesirable. Worth making that explicit in a follow up

Reply:
why's the *_ needed?

Reply:
I think it was just shorthand to not have to type out the class fixtures. Looks to have been introduced in #23544 so cc @gfyoung if there's something I may be overlooking, but I think can type these out explicitly in a follow up PR to avoid injection of undesired fixtures

Reply:
I think this was an inheritance issue, so all of these fixtures were getting passed in.  That's why I added it.  We can revisit though of course.